pkg/chains/objects/objects.go

```diff
@@ -65,6 +68,7 @@ type TektonObject interface {
 	SupportsTaskRunArtifact() bool
 	SupportsPipelineRunArtifact() bool
 	SupportsOCIArtifact() bool
+	SupportsSBOMArtifact() bool
```

Comment:
I know this is following the pattern of existing funcs, but does this make sense to add to the Object interface? 🤔 

I don't know if "whether SBOM artifacts are supported" is a property of an Object we want to enforce on all implementations. Should this be broken out into its own interface?